core-primitives/settings/src/lib.rs

```diff
@@ -98,7 +98,6 @@ pub mod sidechain {
 pub mod enclave {}
 
 /// Settings for the Teeracle
-#[cfg(feature = "teeracle")]
```

Comment:
No real benefit, and this removes the need to use this compiler flag in the runconfig.